### ISSUE TITLE
Cannot contact http://192.168.86.39/

### ISSUE THREAD

**Issue by jameszw001** [yoctopuce-examples/roomba_php#2]
I installed apache2 and PHP 7.2.24. I am getting:

`Cannot contact http://192.168.86.39/`

![image](https://user-images.githubusercontent.com/75647664/110619826-45a4af00-8166-11eb-994f-04a89d750ee8.png)

**Comment by jameszw001** [comment]
seems failing at

```
// Setup the API to use the VirtualHub on local machine,
        if (YAPI::RegisterHub($address, $errmsg) != YAPI_SUCCESS) {
            die("Cannot contact $address");
        }
```

**Comment by jameszw001** [comment]
installed https://www.yoctopuce.com/EN/virtualhub.php but still getting the same error

**Comment by srinsoz** [comment]
This error mean that the PHP server cannot contact the YoctoHub with the IP addresse ``192.168.86.39``. 

Could you double check that the IP address of the YoctoHub plugged on the Roomba is effectively ``192.168.86.39``. If yes are you able to connect to the Web interface of the YoctoHub?

**Comment by jameszw001** [comment]
so I DO need to buy a YoctoHub in order to run your code? I am connecting roomba with a laptop.

**Comment by srinsoz** [comment]
Yes. Our example is designed to use a YoctoHub an a Yocto-Serial.  You should read the full article :

https://www.yoctopuce.com/EN/article/driving-a-roomba-with-a-yoctopuce-module
